model/src/main/java/org/openremote/model/teltonika/PayloadJsonObject.java

```diff
@@ -0,0 +1,49 @@
+package org.openremote.model.teltonika;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
```

Comment:
key ordering in JSON is not guaranteed plus you only have one property being serialised